python/tank/deploy/tank_commands/setup_project_wizard.py

```diff
@@ -641,4 +641,4 @@ def execute(self):
             core_localize.do_localize(self._log, 
                                       self._params.get_configuration_location(sys.platform), 
                                       suppress_prompts=True,
-                                      strip_toolkit_credentials=self._is_session_based_authentication_supported())
```

Comment:
It's not used at the moment, but I thought that maybe we could add a step to the adminui module in the future that allowed you to make that decision.